### DISCUSSION TITLE
Exercise 6, question 4

### DISCUSSION THREAD

**Discussion by EthanJDaniel** [LiNk-NY/IntroR#45] [Q&A]
If I am understanding correctly fitted values are the predicted value based on the linear regression and residual values are the difference between the fitted and the actual.

These are the two graphs (second with the help of chatgpt) I made. I am struggling to understand what exactly they show me, perhaps I made a mistake in the code?

![graph1](https://github.com/CUNYSPHCode/IntroR/assets/170897836/0f2590a4-6afd-4765-ac27-36644bb7d1bb)
![graph2](https://github.com/CUNYSPHCode/IntroR/assets/170897836/8d2d2e93-38f6-4723-9085-015dbfcde5f9)

**Comment by DikiDL** [comment]
Hi Ethan, I also (with the help of chatgpt) got a similar graph but not sure if I'm interpreting it correctly. My understanding is that we have to assess if the residuals are spread out randomly or if there are any trends of patterns to it. If the model is a good fit, then the residual values will be randomly scattered around y=0. In this graph, it looks like it is randomly distributed and there isn't a curve  or trend. 

This is  how I understood it but please let me know if I'm incorrect.

**Reply by ralythompson24** [reply]
I'd say this is a good interpretation of what you're looking for!
